diff --git a/node_modules/@vitejs/plugin-vue/dist/index.js b/node_modules/@vitejs/plugin-vue/dist/index.js
index c15c7ab..8d1d2fe 100644
--- a/node_modules/@vitejs/plugin-vue/dist/index.js
+++ b/node_modules/@vitejs/plugin-vue/dist/index.js
@@ -137,7 +137,7 @@ function setSrcDescriptor(filename, entry, scoped) {
 	cache.set(filename, entry);
 }
 function getHash(text) {
-	return crypto.hash("sha256", text, "hex").substring(0, 8);
+	return crypto.createHash("sha256").update(text).digest("hex").substring(0, 8);
 }
 
 //#endregion
